src/pybind/mgr/dashboard/frontend/src/app/ceph/shared/device-list/device-list.component.html

```diff
@@ -0,0 +1,39 @@
+<cd-table *ngIf="hostname || osdId !== null"
+          [data]="devices"
+          [columns]="columns"></cd-table>
+
+<cd-warning-panel *ngIf="hostname === '' && osdId === null"
+                  type="warning"
+                  i18n>Neither hostname nor OSD ID given</cd-warning-panel>
```

Comment:
I got `cd-warning-panel' is not a known element:` error, maybe it should be `<cd-alert-panel>`?

Reply:
Yes, I think so, too. That's why I added the `needs-rebase` label yesterday.